### ISSUE TITLE
About multi-GPU training

### ISSUE THREAD

**Issue by vesdas** [guoqincode/AnimateAnyone-unofficial#31]
Thank you for your contributions! There are two questions below:

1. I have observed that the training duration using two RTX 6000 ada GPUs exceeds the time it takes with a single GPU. Is this an expected phenomenon?
2. I encountered the phenomenon of gradient explosion during the training process.

**Comment by guoqincode** [comment]
1. To improve GPU utilization, you can change the Dataset to lmdb format or increase the resolution to 512,768.
2. I have not encountered this phenomenon, you can try to use the latest training code, it will still appear, suggested to reduce the learning rate.
